_cultures/communication/slack.md

```diff
@@ -20,15 +21,14 @@ Have in mind who you're trying to reach: If the person you want to correspond wi
 
 Custom display channel: Star the channel heavily used, and only display unread channels if you prefer a clean interface. Click with the left button on the channel and select `Star channel`. 
 <p align="center">
- <img width="200" alt="star" src=https://user-images.githubusercontent.com/53827690/89816335-63e4f680-db14-11ea-85dc-b42ce633a89a.png>
- <img width="400" alt="display" src=https://user-images.githubusercontent.com/53827690/89817201-becb1d80-db15-11ea-9057-3fbd7ac038c0.png>
+    <img width="200" alt="Screen Shot 2021-10-21 at 5 42 44 PM" src="https://user-images.githubusercontent.com/53827690/89816335-63e4f680-db14-11ea-85dc-b42ce633a89a.png">
```

Comment:
just curious about the `alt` change, was that relevant to the fix?

Reply:
Oh no this is my bad, all the imgs were broken so i had to manually copy/paste everything. Forgot to update the alt tags 🙏🏻  will fix

Reply:
and nope not relevant to the fix